Core/HLE/proAdhoc.h

```diff
@@ -793,13 +793,14 @@ class AfterAdhocMipsCall : public PSPAction {
 	AfterAdhocMipsCall() {}
 	static PSPAction* Create() { return new AfterAdhocMipsCall(); }
 	void DoState(PointerWrap& p) override {
-		auto s = p.Section("AfterAdhocMipsCall", 3, 4);
+		auto s = p.Section("AfterAdhocMipsCall", 1, 4);
```

Comment:
Hm, why the reduction here?

Reply:
I'm a bit confused about the versioning on DoState, does the minimum version suppose to start from 1 or should it be started from when it started to be implemented? so i decide to use a much lower value for min ver and do version checking afterwards.

If i created a new PSPAction class (ie. AfterApctlMipsCall) do i start from 5,5 or 1,5 or 1,1 ? so i'm confused with the versioning since i don't know what is the current version to be used as min ver, because other places seems to have higher number (ie. `p.Section("Atrac", 1, 9)`)

Reply:
Oh yeah, your reduction here might be right because the first number is the minimum version you support to load from (often very low since we support save states far back). The second number you increment when you make a change. The returned value you check, so if something wasn't there in v1 but there in v2, you do `if (s >= 2) {  do the stuff }`.  (in `else {}`, it can make sense to check if we're loading ,and if so initialize to some default).